### ISSUE TITLE
missing UCF101-24 in the codebase 

### ISSUE THREAD

**Issue by xinmiaolin** [shinkyo0513/Video-Visual-Explanations#2]
Hi,

You have included results on UCF101-24 in your paper "Towards Visually Explaining Video Understanding Networks with Perturbation". However, in the codebase, there is no option for this dataset, e.g., the argument option pretrain_dataset has no option for UCF101-24. 

It could be really helpful if you add the corresponding code or give a short description of how to use the existing codebase on UCF101-24.

Thanks!

**Comment by shinkyo0513** [comment]
Hi,

I am sorry for the inconvenience about the dataset.
You can access the corrected annotations and the download method for UCF101-24 by referring to this repo: https://github.com/gurkirt/corrected-UCF101-Annots. The videos in the test set used by the paper are listed as follows:
`Basketball/v_Basketball_g01_c07
Basketball/v_Basketball_g06_c04
Basketball/v_Basketball_g03_c06
Basketball/v_Basketball_g06_c02
Basketball/v_Basketball_g04_c04
BasketballDunk/v_BasketballDunk_g07_c01
BasketballDunk/v_BasketballDunk_g06_c02
BasketballDunk/v_BasketballDunk_g05_c04
BasketballDunk/v_BasketballDunk_g04_c03
BasketballDunk/v_BasketballDunk_g04_c02
Biking/v_Biking_g06_c05
Biking/v_Biking_g05_c07
Biking/v_Biking_g01_c01
Biking/v_Biking_g04_c02
Biking/v_Biking_g07_c04
CliffDiving/v_CliffDiving_g02_c01
CliffDiving/v_CliffDiving_g02_c02
CliffDiving/v_CliffDiving_g03_c01
CliffDiving/v_CliffDiving_g05_c03
CliffDiving/v_CliffDiving_g05_c05
CricketBowling/v_CricketBowling_g06_c01
CricketBowling/v_CricketBowling_g02_c04
CricketBowling/v_CricketBowling_g07_c01
CricketBowling/v_CricketBowling_g02_c05
CricketBowling/v_CricketBowling_g05_c01
Diving/v_Diving_g03_c03
Diving/v_Diving_g03_c06
Diving/v_Diving_g06_c04
Diving/v_Diving_g05_c02
Diving/v_Diving_g03_c04
Fencing/v_Fencing_g06_c02
Fencing/v_Fencing_g01_c02
Fencing/v_Fencing_g05_c01
Fencing/v_Fencing_g01_c06
Fencing/v_Fencing_g06_c04
FloorGymnastics/v_FloorGymnastics_g03_c04
FloorGymnastics/v_FloorGymnastics_g04_c02
FloorGymnastics/v_FloorGymnastics_g03_c01
FloorGymnastics/v_FloorGymnastics_g05_c03
FloorGymnastics/v_FloorGymnastics_g07_c07
GolfSwing/v_GolfSwing_g01_c01
GolfSwing/v_GolfSwing_g01_c02
GolfSwing/v_GolfSwing_g02_c01
GolfSwing/v_GolfSwing_g04_c02
GolfSwing/v_GolfSwing_g07_c03
HorseRiding/v_HorseRiding_g03_c07
HorseRiding/v_HorseRiding_g07_c04
HorseRiding/v_HorseRiding_g02_c01
HorseRiding/v_HorseRiding_g02_c02
HorseRiding/v_HorseRiding_g06_c07
IceDancing/v_IceDancing_g02_c06
IceDancing/v_IceDancing_g07_c07
IceDancing/v_IceDancing_g01_c07
IceDancing/v_IceDancing_g04_c06
IceDancing/v_IceDancing_g02_c03
LongJump/v_LongJump_g01_c04
LongJump/v_LongJump_g03_c02
LongJump/v_LongJump_g01_c06
LongJump/v_LongJump_g04_c01
LongJump/v_LongJump_g06_c03
PoleVault/v_PoleVault_g01_c04
PoleVault/v_PoleVault_g02_c01
PoleVault/v_PoleVault_g05_c01
PoleVault/v_PoleVault_g03_c02
PoleVault/v_PoleVault_g03_c07
RopeClimbing/v_RopeClimbing_g06_c01
RopeClimbing/v_RopeClimbing_g07_c05
RopeClimbing/v_RopeClimbing_g02_c05
RopeClimbing/v_RopeClimbing_g04_c01
RopeClimbing/v_RopeClimbing_g03_c02
SalsaSpin/v_SalsaSpin_g04_c01
SalsaSpin/v_SalsaSpin_g05_c01
SalsaSpin/v_SalsaSpin_g01_c02
SalsaSpin/v_SalsaSpin_g07_c04
SalsaSpin/v_SalsaSpin_g06_c04
SkateBoarding/v_SkateBoarding_g02_c03
SkateBoarding/v_SkateBoarding_g02_c02
SkateBoarding/v_SkateBoarding_g07_c02
SkateBoarding/v_SkateBoarding_g05_c04
SkateBoarding/v_SkateBoarding_g06_c03
Skiing/v_Skiing_g06_c02
Skiing/v_Skiing_g03_c02
Skiing/v_Skiing_g04_c04
Skiing/v_Skiing_g05_c02
Skiing/v_Skiing_g02_c04
Skijet/v_Skijet_g02_c03
Skijet/v_Skijet_g06_c02
Skijet/v_Skijet_g04_c02
Skijet/v_Skijet_g06_c03
Skijet/v_Skijet_g03_c03
SoccerJuggling/v_SoccerJuggling_g06_c05
SoccerJuggling/v_SoccerJuggling_g03_c01
SoccerJuggling/v_SoccerJuggling_g03_c04
SoccerJuggling/v_SoccerJuggling_g07_c01
SoccerJuggling/v_SoccerJuggling_g02_c04
Surfing/v_Surfing_g02_c01
Surfing/v_Surfing_g03_c03
Surfing/v_Surfing_g01_c04
Surfing/v_Surfing_g01_c02
Surfing/v_Surfing_g01_c03
TennisSwing/v_TennisSwing_g07_c04
TennisSwing/v_TennisSwing_g04_c02
TennisSwing/v_TennisSwing_g05_c02
TennisSwing/v_TennisSwing_g05_c06
TennisSwing/v_TennisSwing_g06_c06
TrampolineJumping/v_TrampolineJumping_g07_c03
TrampolineJumping/v_TrampolineJumping_g04_c02
TrampolineJumping/v_TrampolineJumping_g05_c04
TrampolineJumping/v_TrampolineJumping_g04_c04
TrampolineJumping/v_TrampolineJumping_g04_c05
VolleyballSpiking/v_VolleyballSpiking_g04_c01
VolleyballSpiking/v_VolleyballSpiking_g05_c04
VolleyballSpiking/v_VolleyballSpiking_g04_c05
VolleyballSpiking/v_VolleyballSpiking_g03_c04
VolleyballSpiking/v_VolleyballSpiking_g06_c03
WalkingWithDog/v_WalkingWithDog_g05_c03
WalkingWithDog/v_WalkingWithDog_g01_c04
WalkingWithDog/v_WalkingWithDog_g06_c05
WalkingWithDog/v_WalkingWithDog_g02_c05
WalkingWithDog/v_WalkingWithDog_g02_c04
`
As mentioned in the experimental setting part, I randomly sampled 16 frames in each video for training networks and sampled fixed 16 frames in each test video. The frames are resized to (112, 112) and normalized by the mean of [0.43216, 0.394666, 0.37645] and the std of [0.22803, 0.22145, 0.216989].

Thank you!

**Comment by xinmiaolin** [comment]
Thank you very much for the detailed answer!
